CONTRIBUTING.md

```diff
@@ -1,5 +1,11 @@
 # Contributing to Node.js
 
+## Code of Conduct
+
+The Code of Conduct explains the *bare minimum* behavior
```

Comment:
Could we move this an the above comment into another PR? Actually changing it is too much of a bikeshed.